opal/test/reachable/Makefile

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2017 Amazon.com, Inc. or its affiliates.  All Rights
+#                    reserved.
+#
+# $COPYRIGHT$
+#
+# Additional copyrights may follow
+#
+# $HEADER$
+#
+
+PROGS = reachable_weighted reachable_netlink
+
+all: $(PROGS)
+
+CC = ortecc
+CFLAGS = -g
+CXX = ortec++
+CXXFLAGS = -g
+FFLAGS = -g
```

Comment:
Nit: are `FFLAGS` and `CXXFLAGS` necessary

Reply:
Sigh :)